### DISCUSSION TITLE
exit node能不能在文档中详细解释一下

### DISCUSSION THREAD

**Discussion by yuyu1025** [EasyTier/EasyTier#459] [General]
我想了解exit node的用法、作用。但是文档对于这个讲的太少了…

**Comment by KKRainbow** [comment]
这个不能细说

**Reply by yuyu1025** [reply]
虽然但是、、、我还是想学一下。我正常配置了exit node，但是没有效果，还有什么其他需要配置的吗？

**Reply by KKRainbow** [reply]
出口节点有设置 --enable-exit-node 么

**Reply by yuyu1025** [reply]
有设置的

**Comment by rainpaper-bs** [comment]
这个是不是涉及到了翻的技术？文档里讲的太少了，真有点看不懂。

**Reply by xkdvSrPD** [reply]
别想了，wireguard流量精确识别秒封的

**Comment by carter115** [comment]
A端：easytier-core -i 10.144.144.1 --enable-exit-node --relay-all-peer-rpc --network-name abc --network-secret abc

B端：easytier-core -i 10.144.144.2 --network-name abc --network-secret abc -p tcp://8.138.xxx.xxx:11010 --exit-nodes 10.144.144.1

**B端连接后，检查自己的IP**
curl https://openapi.lddgo.net/base/gtool/api/v1/GetIp
{"code":0,"msg":null,"data":{"ip":"61.140.xxx.xxx","ipv6":"0000:0000:0000..."}}

**B端手动添加路由**(测试，用来验证自己的出口IP)
route add 47.99.189.0 mask 255.255.255.0 10.144.144.1 metric 1
注：openapi.lddgo.net域名的IP是47.99.189.36

**B端再次检查自己的IP**
curl https://openapi.lddgo.net/base/gtool/api/v1/GetIp
{"code":0,"msg":null,"data":{"ip":"8.138.xxx.xxx","ipv6":"0000:0000:0000..."}}

测试成功

**Reply by uglyfirefish** [reply]
我按照DeepSeek的建议做了一堆设置，结果还是不行，现在运行第一个命令就出错了。

**Reply by gdtv** [reply]
请教一下，
1、B端每访问一个网站都要手动将这个网站的IP添加到路由，这样不现实吧？
2、如果B端是手机，怎样添加路由呢？

**Comment by xkdvSrPD** [comment]
https://tailscale.com/kb/1103/exit-nodes

**Comment by happytrudy** [comment]
拿刀砍人怪卖刀的😁
